content/docs/next-release/features.md

```diff
@@ -55,3 +55,18 @@ from advanced features not available elsewhere, if your organization has already
 using Zipkin libraries, you do not have to rewrite all that code. Jaeger provides backwards compatibility with Zipkin
 by accepting spans in Zipkin formats (Thrift, JSON v1/v2 and Protobuf) over HTTP. Switching from Zipkin backend is just a matter
 of routing the traffic from Zipkin libraries to the Jaeger backend.
+
+## Topology
+
+In production, the Dependencies from `System Architecture` is based on service data which stored in Cassandra or Elasticsearch, 
```

Comment:
Its not specific to production or any particular storage - the main point is that if using production setup then need to also run the spark job, so possibly:

"The 'System Architecture' is based on service dependencies, derived from span data reported to the Jaeger Collector. When using a production configuration, the dependencies are derived using a Spark job."

and then "The 'Deep Dependency Graph' is generated based on trace instances obtained by doing a search. The main differences between them are:"?